server/collab/matchers/__init__.py

```diff
@@ -3,10 +3,12 @@
 from .mnemonic_hash import MnemonicHashMatcher
 from .name_hash import NameHashMatcher
 from .mnemonic_hist import MnemonicHistogramMatcher
+from .pairwise_matcher import PairwiseMatcher
 
 
 matchers_list = [IdentityHashMatcher, NameHashMatcher, AssemblyHashMatcher,
-                 MnemonicHashMatcher, MnemonicHistogramMatcher]
+                 MnemonicHashMatcher, MnemonicHistogramMatcher, PairwiseMatcher]
```

Comment:
`matchers_list` is used to hold only actual matching objects. abstract matching classes (such as `HashMatcher` and the new `PairwiseMatcher`) should not be listed here.

This is the reason you're getting an empty dict for matcher types.